Solutions/Intel471/Playbooks/Intel471-ImportMalwareIntelligenceToSentinel/azuredeploy.json

```diff
@@ -0,0 +1,994 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "metadata": {
+        "title": "Intel 471 Malware Intelligence",
+        "description": "This playbook ingests malware indicators from Intel 471's Titan API into ThreatIntelligenceIndicator table.",
+        "prerequisites": [
+            "1. An active account in Intel 471's Titan platform and API credentials",
+            "2. Storage Account resource and a Container (blob) inside it",
+            "3. Key Vault resource with Titan API credentials stored as Secrets under following keys: TitanUserName and TitanAPIKey",
+            "4. Above resources should be deployed under the same subscription and in the same resource group"
+        ],
+        "postDeployment": [
+            "1. Go to the newly created keyvault API Connection object > Edit API connections > Authorize > Save",
+            "2. Go to created Logic App > Edit and navigate to CollectAndSubmitIndicators > SubmitInBatches > Connections block. Inside this block create a new connection. Select 'Connect with managed identity', provide connection name and click 'Create'",
+            "3. Grant the Logic App `ThreatIndicators.ReadWrite.OwnedBy` role. For more details consult https://learn.microsoft.com/azure/app-service/scenario-secure-app-access-microsoft-graph-as-app#grant-access-to-microsoft-graph"
+        ],
+        "lastUpdateTime": "2023-02-16T00:00:00.000Z",
+        "entities": [
+            "ip",
+            "filehash",
+            "url"
+        ],
+        "tags": [
+            "ThreatIntelligence"
+        ],
+        "author": {
+            "name": "Marcin Molenda / Intel 471, Inc."
+        }
+    },
+    "parameters": {
+        "PlaybookName": {
+            "defaultValue": "intel471-malware-indicators",
```

Comment:
Playbook name should be the one matching with "folder structure" here it should be "Intel471-ImportMalwareIntelligenceToSentinel"